### DISCUSSION TITLE
How to hide the Audio 1/1 and the subtitles button (when no subtitles available )?

### DISCUSSION THREAD

**Discussion by houssemko** [Samillion/ModernZ#221] [Q&A]
I couldn't find a setting in the configuration file .

**Comment by Samillion** [comment]
At the moment the subtitle button will gray out if no subtitles, and both audio and subtitle buttons always show.

 If it's not a bother, could you make a feature request issue with it? To keep track properly.

**Comment by houssemko** [comment]
Okay I will

**Comment by Samillion** [comment]
Completed with https://github.com/Samillion/ModernZ/pull/244
